ui/lib/pki/addon/components/pki-role-form.hbs

```diff
@@ -31,15 +31,75 @@
     <MessageError @errorMessage={{this.errorBanner}} class="has-top-margin-s" />
     {{! ARG TODO write a test for namespace reminder }}
     <NamespaceReminder @mode={{if @model.isNew "create" "update"}} @noun="PKI role" />
-    <FormFieldGroupsLoop
-      @model={{@model}}
-      @mode={{if @model.isNew "create" "update"}}
-      @modelValidations={{@modelValidations}}
-      @showHelpText={{false}}
-      as |attr|
-    >
-      <RadioSelectTtlOrString @model={{@model}} @attr={{attr}} />
-    </FormFieldGroupsLoop>
+    {{#each @model.fieldGroups as |fieldGroup|}}
```

Comment:
ended up not being able to use `FormFieldGroupsLoop` for two main reasons:
1. because there is a lot of custom work required to make the CSS grid for Key Usage work, and it didn't make sense to apply that logic to a widely used component where PKI might be a unique situation where CSS grid is needed.
2. I needed to pass in a conditional attr when dealing with key_bits, this complicated a whole mess of things and again is a one off situation specific to pki.